### ISSUE TITLE
eXist 4.5 using local catalogs sending large amounts of outbound data

### ISSUE THREAD

**Issue by lopsotronic** [eXist-db/exist#2476]
Thank you for reporting your issue!

To be able to better understand you problem or suggestion, please add as much information as possible to this ticket. When there is no or just little information, it will be too difficult to work on it (and hence we will not be able to solve it)

Please fill in the following sections:

### What is the problem
eXist 4.5 with XML files loaded (S1000D issue 3.0.1 schema). On login, catalog view, and xquery, outbound network activity totaling more than 25 gb to s1000d.org and/or publications.airlines.org. 

### What did you expect
S1000D issue 3.0.1 schemas are loaded on the local catalog with all required entity files. There should be *no* outbound or inbound network activity from eXist process.

### Describe how to reproduce or add a test
Usage of eXist 4.5 with S1000D 3.0.1 "Bicycle test" publicly available sample XML files using a local catalog. Check for outbound network activity on load, catalog view, and any xquery activity. Note that S1000D does not use namespaces and includes a doctype declaration. Typical root level with doctype appears as follows

----
<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE dmodule [
	<!ENTITY ICN-S1000DBIKE-AAA-DA23000-0-U8025-00532-A-04-1 SYSTEM "../../../illustrations/ICN-S1000DBIKE-AAA-DA23000-0-U8025-00532-A-04-1.CGM" NDATA cgm>
]>
<dmodule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.s1000d.org/S1000D_3-0/xml_schema_flat/proced.xsd" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dc="http://www.purl.org/dc/elements/1.1/">
----

Schema catalog includes
http://www.s1000d.org/S1000D_3-0/xml_schema_flat/descript.xsd
http://www.s1000d.org/S1000D_3-0/xml_schema_flat/proced.xsd
http://www.s1000d.org/S1000D_3-0-1/xml_schema_flat/appliccrossreftable.xsd
http://www.s1000d.org/S1000D_3-0-1/xml_schema_flat/crew.xsd
http://www.s1000d.org/S1000D_3-0-1/xml_schema_flat/descript.xsd
http://www.s1000d.org/S1000D_3-0-1/xml_schema_flat/ipd.xsd
http://www.s1000d.org/S1000D_3-0-1/xml_schema_flat/proced.xsd
http://www.s1000d.org/S1000D_3-0-1/xml_schema_master/dm/crewSchema.xsd

### Context information
Please always add the following information
2019-02-05 11:03:22,842 [global.launcher.startJetty] INFO  (JettyStart.java [run]:158) - [eXist Version : 4.5.0] 
2019-02-05 11:03:22,842 [global.launcher.startJetty] INFO  (JettyStart.java [run]:159) - [eXist Build : 201811211859] 
2019-02-05 11:03:22,842 [global.launcher.startJetty] INFO  (JettyStart.java [run]:160) - [Git commit : e29b4099c] 
2019-02-05 11:03:22,842 [global.launcher.startJetty] INFO  (JettyStart.java [run]:162) - [Operating System : Windows Server 2016 10.0 amd64] 
- Windows installer

[S1000D301_BikeSample.zip](https://github.com/eXist-db/exist/files/2854263/S1000D301_BikeSample.zip)
[S1000D301_schema.zip](https://github.com/eXist-db/exist/files/2854264/S1000D301_schema.zip)
[S1000D301_BikeSample.zip](https://github.com/eXist-db/exist/files/2854268/S1000D301_BikeSample.zip)
[S1000D301_schema.zip](https://github.com/eXist-db/exist/files/2854269/S1000D301_schema.zip)

- No custom changes

**Comment by dizzzz** [comment]
Please could the whole template? There is information in it which I probably need.

**Comment by lopsotronic** [comment]
Unable to provide catalog in use, as it contains sensitive data. Outbound traffic detected by third party systems. S1000D301_BikeSample.zip contains publicly available examples using the same schema as our catalog and will be representative of our catalog. Question: is "details how you configured eXist-db to use te (sic) catalog file" captured in any configuration file on the application server? If yes, I can scan that file for sensitive information and, if not containing sensitive information, send that to this issue thread.

**Comment by adamretter** [comment]
@lopsotronic The XML files that you provided make use of the Dublin Core namespace. Could this be the thing that is generating the traffic?

Can you please use Wireshark (or another similar tool of your choice) in the first instance, to find out where this traffic inbound/outbound traffic is directed, that should give you some idea of what is being requested over the net.

**Comment by duncdrum** [comment]
@lopsotronic since there hasn't been a response, thee is little we can do in trying to debug this. To keep this ticket open, please provide us with some information about the outgoing traffic destination.

**Comment by adamretter** [comment]
I note that eXist-db 6 has a new Catalog Resolver which may be helpful

**Comment by line-o** [comment]
This should be retested.

**Comment by dizzzz** [comment]
I agree. A catalog file would help a bit. Assumptions.... XSDs and catalogs are stored on disk?
I completely forgot about this report....
